src/dropdown/list/_dropdown-list.scss

```diff
@@ -47,7 +47,3 @@
   visibility: visible;
   opacity: 1;
 }
-
-.dropdown-empty-message {
-  padding: 0 15px;
-}
```

Comment:
This rule is removed, but it wasn't added with the new name (`dropdown-list__empty-message`)
Is this rule not necessary anymore?

Reply:
Nope there is no need to give it a padding anymore